src/lib/mina_stdlib/time.ml

```diff
@@ -0,0 +1,24 @@
+open Core_kernel
+
+module Span = struct
+  [%%versioned
+  module Stable = struct
+    module V3 = struct
+      type t = Core_kernel.Time.Stable.Span.V3.t [@@deriving sexp]
```

Comment:
Nit: would be nice to have a simple line saying in which format the time is given (us, ns, etc). And have some consistency checks (can be a low priority follow-up).